src/plugins/GreaseMonkey/gm_script.cpp

```diff
@@ -148,6 +152,31 @@ QWebEngineScript GM_Script::webScript() const
     return script;
 }
 
+bool GM_Script::isUpdating()
+{
+    return m_updating;
+}
+
+void GM_Script::updateScript()
+{
+    if (!m_downloadUrl.isValid() || m_updating)
+        return;
+
+    m_updating = true;
+    emit updatingChanged(m_updating);
+
+    GM_Downloader* downloader = new GM_Downloader(m_downloadUrl, m_manager);
+    downloader->updateScript(m_fileName);
+    connect(downloader, &GM_Downloader::finished, this, [=](){
```

Comment:
You only need to capture `ŧhis` here, so please change it. Same in the connect bellow.